### ISSUE TITLE
Dropdown and drawer components do not works when the table is paginated

### ISSUE THREAD

**Issue by alkadoHs** [themesberg/flowbite#699]
I'm using data tables to enhance pagination, filtering and searching, but when the table is paginated the dropdowns and drawers of the next page stops to work and those on the first page works fine.
I don't understand why this issue is happening.

**Comment by zoltanszogyenyi** [comment]
Hey @alkadoHs,

Did you make sure that the IDs of these dropdowns and modals are unique even across paginations?

Cheers,
Zoltan

**Comment by alkadoHs** [comment]
> Hey @alkadoHs,
> 
> Did you make sure that the IDs of these dropdowns and modals are unique even across paginations?
> 
> Cheers,
> Zoltan

Yes, they are unique, I'm looping through the data from the server and providing each modal and it's corresponding trigger button with unique id of the product. But it doesn't work 😔

**Comment by zoltanszogyenyi** [comment]
@alkadoHs what framework are you using?

**Comment by alkadoHs** [comment]
> @alkadoHs what framework are you using?

Codeigniter, a PHP framework

**Comment by zoltanszogyenyi** [comment]
Do you have an example online somewhere? It's hard to guess why it wouldn't work.

Did you check the `FlowbiteInstances` object to see what's going on?

**Comment by xujiongze** [comment]
Same problem
Im using laravel10 + vue3 + inertia
In inertia, using <Link> insted <a> "This component is a light wrapper around a standard anchor <a> link that intercepts click events and prevents full page reloads"
And all page opened with Link, can't use offcanva(Drawer)

**Comment by embarekmessai** [comment]
> ## @zoltanszogyenyi
> Same problem
> Im using laravel10 + vue3 + inertia
> In inertia, using insted "This component is a light wrapper around a standard anchor link that intercepts click events and prevents full page reloads"
> And all page opened with Link, can't use offcanva(Drawer)
> Solution for inertia:
> 
> import { router } from '@inertiajs/vue3'
> 
> router.on('success', () => { initFlowbite(); })

Thank you, this helps me
